pipeline-module/README.md

```diff
@@ -1,3 +1,131 @@
+- Please remember to make it private.
+
+### 4. Push the local Git repository to GitHub
+
+Back in your shell follow the directions GitHub provides to push your existing repository to to the GitHub repository you just created.
+The instructions look like this:
+
+```shell
+git remote add origin https://github.com/SPHEREx/ssdc-ms-<...>.git
+git branch -M main
+git push -u origin main
+```
+
+**The GitHub instructions are customized to the name of your specific repository, so feel free to copy and paste from the GitHub page.**
+
+### 5. Configure the repository on GitHub
+
+Once you've pushed the initial content to GitHub you can begin configuring it:
+
+- From the **About** section on the main page of the repository, click on the :gear: (gear) icon and add the `ssdc-docs` “topic” to the repository.  (This will help us [navigate the many repos](https://github.com/search?q=org%3ASPHEREx+ssdc-docs&type=repositories) that we’ll have on this organization.)
+
+Let Gregory know at least once a day that you’ve created new repos, and he’ll set the access permissions appropriately to make sure that the whole team has access.
+
+### 6. Check the initial build
+
+About 2–3 minutes after pushing the repository to GitHub, the initial build should be ready.
+On the repository's GitHub page, check the build to ensure it worked:
+
+- Check on the **Actions** tab.
+- There should be an item in the table corresponding to the name of the initial commit. The :white_check_mark: (green check mark) icon shows that the build completed.
+- Click on the title to see the build's page. From this page, you can download the PDF "artifact."
+
+If the build didn't work, ask for help.
+
+Check on the GitHub site for the repository to make sure that the initial “Actions” have completed successfully.  Ask for help if they don’t.
+
+### 7. Add the initial content
+
+If all is well, continue working on on your local repository clone to add the actual module text and any figure files that are needed.
+Start by creating a branch for this work:
+
+```shell
+git checkout -b initial-text
+```
+
+Then copy over the content from the original document.
+Replace any content **between** `\maketitle` and `\bibliography` with the original document's content.
+**Leave the \bibliography{spherex} from the template rather than using the one from the original document.**
+
+[Learn more about the spherex document class and spherex bibliography file.](https://github.com/SPHEREx/spherex-tex)
+
+Copy any images from the original document on Overleaf into the new repo.
+
+If the original document used `\modulename` in the body of the document to substitute in the name of the module, you will need to use `\themodulename` for this instead.
+(`\modulename{name}` is now the setter for the name, rather than the getter.)
+
+If you'd like, you can compile the document locally.
+See the [spherex-tex](https://github.com/SPHEREx/spherex-tex) repository's for instructions.
```

Comment:
See the [spherex-tex](https://github.com/SPHEREx/spherex-tex) repository's [README](https://github.com/SPHEREx/spherex-tex/blob/main/README.md) for instructions.

... was what I had in mind.

Reply:
Sorry about that, got the README link in now.